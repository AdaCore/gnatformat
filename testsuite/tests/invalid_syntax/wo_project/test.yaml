description:
  Test that syntax errors are greacefully handled
driver: gnatformat
args: ["--pipe", "test.adb", "test_a.ads", "test_dep.ads"]
status_code: 1
