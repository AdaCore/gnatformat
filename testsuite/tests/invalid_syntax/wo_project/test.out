--  test.adb
with Test_A;
with Test_Dep;

procedure Main is
begin
   null;
end Main;

--  test_a.ads failed to format
2:4: Cannot parse <sub_object_decl>
2:61: Expected ';', got ')'
2:61: Skipped token )
2:62: Skipped token ;
3:5: Missing ';'
3:5: Skipped token package
3:13: Skipped token Test_A
3:19: Skipped token ;
